src/graphql.tsx

```diff
@@ -52,6 +65,25 @@ export declare interface QueryOptions {
   skip?: boolean;
 }
 
+export interface GraphQLDataProps {
+  error?: ApolloError;
+  networkStatus: number;
+  loading: boolean;
+  variables: {
+    [variable: string]: any;
+  };
+  fetchMore: (fetchMoreOptions: FetchMoreQueryOptions & FetchMoreOptions) => Promise<ApolloQueryResult>;
+  refetch: (variables?: any) => Promise<ApolloQueryResult>;
+  startPolling: (pollInterval: number) => void;
+  stopPolling: () => void;
+  subscribeToMore: (options: SubscribeToMoreOptions) => () => void;
+  updateQuery: (mapFn: (previousQueryResult: any, options: UpdateQueryOptions) => any) => void;
+}
+
+export interface InjectedGraphQLProps<T> {
+  data?: T & GraphQLDataProps;
```

Comment:
this is fine IMO, because GraphQLDataProps is also exported. that way the user can just make their own interface i.e.

```ts
import { GraphQLDataProps } from 'react-apollo';

interface MyCustomInterface<T> {
  whatever?: T & GraphQLDataProps
}
```